src/webswview/lkswview/api/views.py

```diff
@@ -16,4 +25,32 @@ def get(self, id):
         return {'task': 'Say "Hello, World!"'}
 
 
+class SrcPackage(Resource):
+    def get(self, repo_name, suite_name, name):
+        with session_scope() as session:
+            rss = repo_suite_settings_for(session, repo_name, suite_name, fail_if_missing=False)
+            if not rss:
+                abort(404)
+
+            spkgs = (
+                session.query(SourcePackage)
+                .options(undefer(SourcePackage.version))
+                .filter(SourcePackage.repo_id == rss.repo_id)
+                .filter(SourcePackage.suites.any(ArchiveSuite.id == rss.suite_id))
+                .filter(SourcePackage.name == name)
+                .filter(SourcePackage.time_deleted.is_(None))
+                .order_by(SourcePackage.version.desc())
+                .all()
+            )
+            if not spkgs:
+                abort(404)
+
+            return {
+                'pkgs': spkgs,
+                'pkg_repo': rss.repo,
+                'pkg_suite': rss.suite,
+            }
+
+
 api_wrap.add_resource(TodoItem, '/todos/<int:id>')
+api_wrap.add_resource(SrcPackage, '/src/<repo_name>/<suite_name>/<name>')
```

Comment:
```suggestion
api_wrap.add_resource(SrcPackage, '/package/src/<repo_name>/<suite_name>/<name>')
```
Should likely have a package prefix, as we may add more API later.